.github/actions/expo-caches/action.yml

```diff
@@ -203,28 +203,28 @@ runs:
       if: inputs.hermes-engine-aar == 'true'
       uses: actions/cache@v2
       with:
-        path: android/ReactAndroid/prebuiltHermes
-        key: hermes-engine-aar-v2-${{ hashFiles('android/sdks/.hermesversion') }}
+        path: android/prebuiltHermes
+        key: hermes-engine-aar-v2-${{ hashFiles('react-native-lab/react-native/sdks/.hermesversion') }}
     - name: Check hermes-engine cache-hit
       if: inputs.hermes-engine-aar == 'true'
       id: cache-hermes-engine-aar
       shell: bash
       run: |
-        CURRENT_VERSION=$(test -f android/ReactAndroid/prebuiltHermes/.hermesversion && cat android/ReactAndroid/prebuiltHermes/.hermesversion) || true
-        TARGET_VERSION=$(test -f android/sdks/.hermesversion && cat android/sdks/.hermesversion) || true
+        CURRENT_VERSION=$(test -f android/prebuiltHermes/.hermesversion && cat android/prebuiltHermes/.hermesversion) || true
+        TARGET_VERSION=$(test -f react-native-lab/react-native/sdks/.hermesversion && cat react-native-lab/react-native/sdks/.hermesversion) || true
         if [[ $CURRENT_VERSION == $TARGET_VERSION ]]; then echo '::set-output name=cache-hit::true'; else echo '::set-output name=cache-hit::false'; fi
     - name: 🛠 Rebuild hermes-engine AAR when cache missed
       if: inputs.hermes-engine-aar == 'true' && inputs.rebuild-hermes-engine-aar-if-needed == 'true' && steps.cache-hermes-engine-aar.outputs.cache-hit != 'true'
       shell: bash
       run: |
         # Ensure node packages are installed when yarn-workspace cache miss
         yarn install --frozen-lockfile
-        mkdir -p ReactAndroid/prebuiltHermes
-        cp -f sdks/.hermesversion ReactAndroid/prebuiltHermes/
+        mkdir -p prebuiltHermes
+        cp -f ../react-native-lab/react-native/sdks/.hermesversion prebuiltHermes/
         ./gradlew :ReactAndroid:hermes-engine:assembleRelease
         ./gradlew :ReactAndroid:hermes-engine:assembleDebug
-        cp -f ReactAndroid/hermes-engine/build/outputs/aar/hermes-engine-release.aar ReactAndroid/prebuiltHermes/
-        cp -f ReactAndroid/hermes-engine/build/outputs/aar/hermes-engine-debug.aar ReactAndroid/prebuiltHermes/
+        cp -f ../react-native-lab/react-native/ReactAndroid/hermes-engine/build/outputs/aar/hermes-engine-release.aar prebuiltHermes/
+        cp -f ../react-native-lab/react-native/ReactAndroid/hermes-engine/build/outputs/aar/hermes-engine-debug.aar prebuiltHermes/
```

Comment:
Small downside of using the fork in CI jobs is that it now needs to checkout submodules too 🤔 It's not a big problem, but I just wanted to note that.

Reply:
yes, previously i want to do the building from react-native fork in sdk 46, and it broke classic build because turtle-v1 doesn't clone submodules. i think doing this will benefit expo/expo repo healthy and pr reviewing. remembering [this review comment](https://github.com/expo/expo/pull/16752#pullrequestreview-932542408) 😛